R/rlistings_methods.R

```diff
@@ -49,8 +57,14 @@ format_colvector <- function(df, colnm, colvec = df[[colnm]]) {
     strvec
 }
 
+#' @rdname listing_methods
+#' @param vec A vector.
+#' @export
```

Comment:
If this were going to be exported I would expect it to also be moved to `formatters` (which it may eventually need to be anyway)

Reply:
Should I just remove the export on line 62? Or is the best practice to remove all of the roxygen lines?

Reply:
just the export is fine, though in this case the other lines aren't doing very much in practice...